### PR TITLE
Kaoru style up

### PR DESCRIPTION
**Pull request by KaoruDev**

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -1,7 +1 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+User.create(email: 'user@mail.com', password: 'password')
```
Comment (gkop) [inline db/seeds.rb]: seeds.rb should be idempotent and use domains from https://tools.ietf.org/html/rfc2606#page-3

**Comment by gkop** [comment]
I'm worried about tracking upstream changes from https://github.com/18F/web-design-standards . Could we use their official npm package with npm shrinkwrap to facilitate staying in sync? Perhaps symlink node_modules/ to vendor/assets/ ?

Also would you check that we aren't pulling in jQuery twice with jquery-rails and from the US design libs? We should only pull the version provided by the US design libs...

**Comment by KaoruDev** [comment]
Rails doesn't really provide a clear way of managing `node_modules`, a symlink doesn't feel like the right answer since we'd still have to copy over the fonts ( typically done with a gulp task ).

Right now to update we have to `npm install` `cp -r node_modules/uswd/... app/assets/stylesheets/vendors/uswd` and do the same for fonts. this also allows us to be selective in which assets to copy over. We actually are not using `uswds` javascript lib yet.

**Comment by gkop** [comment]
That makes sense, especially for this simple prototype.

:shipit:

**Pull request merged**